src/Plugins/jquery.calendar.js

```diff
@@ -1661,7 +1661,11 @@
             var left = offsetMe.left;
 
             var daystr = this.abbr;
-            var day = strtodate(daystr+" 00:00");
+            //***Andres Chabur
+//            var day = strtodate(daystr+" 00:00");
+            var arrdays = daystr.split('/');
```

Comment:
The implementation is incorrect. The user date format may be like 2017-12-12. What will be the result of your `arrdays` then

Reply:
daystr comes with 2017/12/12 date format so then ...
var arrdays = daystr.split('/'); --- --- 
this is the result
arrdays[0] == 2017
arrdays[1] == 12
arrdays[2] == 12

Reply:
you missing the `-` character. as you have assumed the separator will be '/'. there is a configuration option to provide the seperator. you should use that instead of using hardcoded '/'

Reply:
ok , thanks yo so much, i apreciate your comment, i will change it